packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -16,6 +16,7 @@ export interface PreferenceContribution {
 export const PreferenceSchema = Symbol("PreferenceSchema");
 
 export interface PreferenceSchema {
+    name: string,
```

Comment:
Instead of adding a group label, we should make use of the convention to use dot-separated property names. 
The tree should be based on the splitted segments. This will allow extensions to contribute preferences to arbitrary groups, too.